setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I guess its fine to have a bash setup script. 

Normally I'd expect solutions to have a readme with prerequisites section (with relevant links) and Getting Started section. A good place to explain the purpose of command line args if you decide to make use of them (for credentials/settings)

Reply:
If you agree about having README.md and command line args rather than bash setup and a config file I'd suggest its out of scope for this PR. You can add a task in trello for that.

Reply:
Okay yeah that's a fair point, I'll add some more detailed instructions to the existing readme and look into using command line arguments instead